### DISCUSSION TITLE
.NET - Show HTTP status code in response

### DISCUSSION THREAD

**Discussion by MiddleKerb** [scalar/scalar#3096] [Q&A]
How to show the HTTP status code in test request response modal, like in scalar [examples](https://docs.scalar.com/swagger-editor#tag/planets/GET/planets)
![image](https://github.com/user-attachments/assets/a95e1fb7-9c0c-48e7-aa3d-b054b5853900)

mine only returned response time
![image](https://github.com/user-attachments/assets/a5525a36-abb9-4689-90c1-0027b436152c)

config
```c#
if (app.Environment.IsDevelopment())
{
    app.UseSwagger(options =>
    {
        options.RouteTemplate = "openapi/{documentName}.json";
    });
    app.MapScalarApiReference(options =>
    {
        options
            .WithTitle("Example API")
            .WithTheme(ScalarTheme.Kepler)
            .WithDefaultHttpClient(ScalarTarget.Shell, ScalarClient.Curl);
        options.HideModels = true;
        options.ForceThemeMode = Scalar.AspNetCore.ThemeMode.Dark;
        options.DarkMode = true;
    });
}
```

**Comment by hanspagel** [comment]
Wait, that should work! I’ll create an issue.

**Comment by MiddleKerb** [comment]
https://github.com/scalar/scalar/issues/3097#issuecomment-2383432932
